enterprise/utils.py

```diff
@@ -1931,21 +1951,28 @@ def enroll_licensed_users_in_courses(enterprise_customer, licensed_users_info, d
         'pending': [],
         'failures': [],
     }
-    for licensed_user_info in licensed_users_info:
-        user_email = licensed_user_info.get('email')
-        course_mode = licensed_user_info.get('course_mode')
-        course_run_key = licensed_user_info.get('course_run_key')
-        license_uuid = licensed_user_info.get('license_uuid')
-        activation_link = licensed_user_info.get('activation_link')
-
-        user = User.objects.filter(email=licensed_user_info['email']).first()
+    for subsidy_user_info in subsidy_users_info:
+        user_email = subsidy_user_info.get('email')
```

Comment:
do we have any best practices here regarding handling email user input? such as always down-casing the input email string? perhaps stripping leading/trailing white space? looks like we just pass it straight through to a filter later.

Reply:
noooot that I know of in terms of BP but I can def sanitize